client_plugin/Makefile

```diff
@@ -361,11 +361,11 @@ test-e2e:
 .PHONY: test-e2e-runalways test-e2e-runonce
 test-e2e-runalways:
 	$(log_target)
-	$(GO) test -v -timeout 30m -tags runalways $(E2E_Tests)
+	$(GO) test -v -timeout 40m -tags runalways $(E2E_Tests)
```

Comment:
Hit the test timeout atleast twice, there are are too many tests to fit within 30m anymore.

We could cut back on extra verifications in the code - like verifying volume status after each step in several tests. There can be one test which verifies all this (status of a volume after attach/detach/, but every test doing the same may be too much.